pallets/offchain-worker/src/lib.rs

```diff
@@ -62,16 +147,16 @@ decl_storage! {
 		TotalClaims get(fn total_claims): u64;
 		/// Record the accounts send claims in latest block
 		ClaimAccountSet get(fn query_account_set): map hasher(blake2_128_concat) T::AccountId => ();
-		/// Record account's ethereum balance
-		AccountBalance get(fn account_balance): map hasher(blake2_128_concat) T::AccountId => u64;
+		/// Record account's btc and ethereum balance
+		AccountBalance get(fn account_balance): map hasher(blake2_128_concat) T::AccountId => (u128, u128);
```

Comment:
Currently it may be fine. But later when we have more and more functionalities, it is not possible to have them all together, even only the balance. It is possible to store them together (but make it more hard to understand the get function)  , but to fetch them together may be a bad idea.
Besides, I recommend using `Options<u128>` and will make later logic a bit easier

Reply:
A good advice. I noticed it and it would be updated as what you mentioned here.